lib/buyers/models.py

```diff
@@ -62,3 +64,29 @@ def incr_lockout(self):
 
     def get_uri(self):
         return reverse('generic:buyer-detail', kwargs={'pk': self.pk})
+
+
+class PaymentMethod(Model):
+    active = models.BooleanField(default=True)
+    buyer = models.ForeignKey(Buyer)
+    provider = models.PositiveIntegerField(choices=PROVIDERS_CHOICES,
+                                           db_index=True)
+    # An id specific to the provider.
+    provider_id = models.CharField(max_length=255)
+    # The type of payment method eg: card or carrier billing.
+    type = models.PositiveIntegerField(choices=(
+        (PAYMENT_METHOD_CARD, PAYMENT_METHOD_CARD),
+        (PAYMENT_METHOD_OPERATOR, PAYMENT_METHOD_OPERATOR),
+    ))
+    # Details about the type, eg Amex, Orange.
+    type_name = models.CharField(max_length=255)
+    # For credit cards, this is the last 4 numbers, could be a truncated
+    # phone number or paypal account for example.
+    truncated_id = models.CharField(max_length=255)
+
+    class Meta(Model.Meta):
+        db_table = 'buyer_payment_method'
```

Comment:
where is ``buyer_payment_method`` defined? All I see is a ``braintree_pay_method`` table